src/main/scala/com/softwaremill/realworld/articles/ArticlesEndpoints.scala

```diff
@@ -50,6 +50,29 @@ class ArticlesEndpoints(articlesService: ArticlesService, base: BaseEndpoints):
           .pipe(defaultErrorsMappings)
     )
 
+  val feedArticles: ZServerEndpoint[Any, Any] = base.secureEndpoint.get
+    .in("api" / "articles" / "feed")
+    .in(
+      query[Int]("limit")
```

Comment:
the definition of the limit paramter is repeated from above - can be extracted to a value